filebeat/inputsource/udp/server_test.go

```diff
@@ -0,0 +1,75 @@
+package udp
+
+import (
+	"net"
+	"runtime"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+)
+
+const maxMessageSize = 20
+const timeout = time.Second * 15
+
+type info struct {
+	message []byte
+	addr    net.Addr
+}
+
+func TestReceiveEventFromUDP(t *testing.T) {
+	if runtime.GOOS == "window" {
+		t.Skipf("Skipped on windows")
```

Comment:
What was the failure that led to this? I'm not a fan of doing this because it could be masking issues with the code or the test.